<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kryptal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="FileSystemList" name="fileSystemList"/>
    </item>
    <item>
     <widget class="QPushButton" name="createFilesystemButton">
      <property name="text">
       <string>Create Filesystem</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="aboutButton">
      <property name="text">
       <string>About</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSystemList</class>
   <extends>QListWidget</extends>
   <header>kryptal/gui/view/widgets/FileSystemList</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
